test-e2e/android/app/src/androidTest/java/com/mopro/mopro_app/ExampleInstrumentedTest.kt

```diff
@@ -42,11 +43,15 @@ class ExampleInstrumentedTest {
         composeTestRule.onNodeWithTag("circomGenerateProofButton").performClick()
         composeTestRule.onNodeWithTag("circomGenerateProofButton").assertIsDisplayed()
 
-        // Test click circom verify proof button
-        // Wait until the second button is enabled
-        Thread.sleep(500)
-
-        composeTestRule.onNodeWithTag("circomVerifyProofButton").performClick()
+        // Wait up to 5 seconds for any circomVerifyProofButton to become enabled before continuing the test
+        composeTestRule.waitUntil(timeoutMillis = 5000) {
+            try {
+                composeTestRule.onNodeWithTag("circomVerifyProofButton").assertIsEnabled()
```

Comment:
and right now the `VerifyProofButton` is not disabled
so we cannot use the `assertIsEnabled()` to check

Reply:
It took me a while to find this fix. If it doesn't work, I'll find another time to address this. Rolled back the changes for now.